dashboard/lib/projects_list.rb

```diff
@@ -121,48 +121,41 @@ def fetch_published_projects(project_group, limit:, published_before: nil)
       end
       if project_group == 'all'
         raise ArgumentError, 'Cannot specify published_before when requesting all project types' if published_before
-        return include_featured(limit: limit)
+        fetch_published_project_types(PUBLISHED_PROJECT_TYPE_GROUPS.keys, limit: limit)
       end
       raise ArgumentError, "invalid project type: #{project_group}" unless PUBLISHED_PROJECT_TYPE_GROUPS.key?(project_group.to_sym)
       fetch_published_project_types([project_group.to_sym], limit: limit, published_before: published_before)
     end
 
-    def include_featured(limit:)
```

Comment:
This is unnecessary because featured projects are published.

Reply:
Nice simplification!